encoding/jsonbq/marshal.go

```diff
@@ -0,0 +1,47 @@
+package jsonbq
+
+import (
+	"cloud.google.com/go/bigquery"
+	"encoding/json"
+)
+
+const (
```

Comment:
Add a comment about these constants above `const`

Reply:
Added a comment. Do you think that description is sufficient?